docs/en/observability/synthetics-recorder.asciidoc

```diff
@@ -0,0 +1,84 @@
+// Content type: Reference(ish)
+//
+// Purpose: Provide complete overview of the Synthetics Recorder UI.
+//
+// Structure: Reflects the pieces of the UI. Also somewhat sequential.
+//
+// Audience assumptions:
+// - The reader may or may not be familiar with Elastic Synthetics already.
+// - The reader might have come here from "Write a synthetic test" or they might have
+// arrived here from a search without larger no context.
+// - The reader might be using Project monitors, Heartbeat, or Elastic Agent.
+// 
+// What's not included:
+// - Detailed definitions of journey, step, etc.
+// - Detailed instructions on how to create a monitor. Point to instructional guide instead.
+
+[[synthetics-recorder]]
+= Synthetics Recorder
+
+You can use the Synthetics Recorder to <<synthetics-create-test>> by interacting with a web page and exporting journey code that reflects all the actions you took.
+
+[discrete]
+== Set up
+
+// Rough requirements to cover
+Requirements:
+* **Elastic Synthetics Recorder**. Download and install the Synthetics Recorder from the release page.
+* **Chrome**. Download and install the Chrome browser. This will be needed to ...
```

Comment:
The browser is bundled together with the Recorder so users won't need to download it separately.